docs-src/docs/kognic-io/resources/pointclouds.md

```diff
@@ -4,7 +4,9 @@ title: Point clouds
 
 The API allows uploading of annotation project related data such as images and point clouds. Kognic uses a potree format internally to represent and present point clouds, this means that uploaded point cloud data needs to be converted into this format before it can be used as input in the system. We currently support automatic conversion of three formats: **pcd**, **csv** and **las**. The converter does not however exhaustively support all possible versions of these formats, see below for details of each format.
 
-It might also be worth noticing that timestamp, color, and other auxillary data is currently discarded in the conversion. Timestamps because our potree version does not support timestamps, and color because it is currently not used in the system.
+A timestamp field must always be present in pointclouds, both in single-frame and sequence inputs, but the values are irrelevant [if motion compensation is not enabled](../inputs/lidars_with_imu_data.md#enabledisable-motion-compensation).
+
+Color and other auxiliary data that is not used in the platform is currently discarded in the conversion to potree.
```

Comment:
I don't think that the intensity field is discarded. It would be great if you could verify this and update this line if needed